rules/node.libsonnet

```diff
@@ -35,12 +35,19 @@
               ))
             ||| % $._config,
           },
-          // Add separate rules for Free, so we can aggregate across clusters in dashboards.
-          // TODO: adjust recording rule name to match best practices
+          // Add separate rules for Available memory, so we can aggregate across clusters in dashboards.
```

Comment:
Don't feel too strongly, but could you add the comment about older kernels here? Just for future me trying to understand why we don't just use `node_memory_MemAvailable_bytes` everywhere :)

Reply:
Oh, sure, well, it's merged now. :laughing: 

For the record, MemAvailable requires Linux >= 3.14.

Reply:
Yeah only realized after merging, could you do a follow up? :)